<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>1cc8d4a6a2887211353d7d194ad2ab92.jpg</normaloff>1cc8d4a6a2887211353d7d194ad2ab92.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="main_pushButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>290</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Розпочати</string>
    </property>
   </widget>
   <widget class="QPushButton" name="main_pushButton_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>290</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Завершити</string>
    </property>
   </widget>
   <widget class="QLabel" name="main_window">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Розпочати роботу програми?</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>615</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
